### PR TITLE
Add section about log forwarding level mapping

### PR DESCRIPTION
**Pull request by sivachandran**
## What's changed

Adds a note about how log levels are mapped in log forwarding

### CODE REVIEW

File: docs/src/increase-observability/logs/forward-logs.md [hunk 1]
```diff
@@ -24,6 +24,14 @@ To enable log forwarding in a project, you need to be a [project admin](../../ad
 You also need your project to have the capability for log forwarding.
 To get that, contact [support](https://console.platform.sh/-/users/~/tickets/open).
 
+{{< note >}}
+
+The log forwarding feature currently does not preserve the original log level. However, to preserve the original log level, there are two alternatives:
```
Comment (ricardokirkner) [inline docs/src/increase-observability/logs/forward-logs.md:29]: `By default, the log forwarding feature will log everything at the INFO level, disregarding the original level used. In order to preserve the original log level you can use the language specific syslog module/package for logging.`

Maybe add some examples of how you'd do this for PHP and Python?

File: docs/src/increase-observability/logs/forward-logs.md [hunk 2]
```diff
@@ -125,3 +125,64 @@ To include a property, add it as a flag, for example `--protocol tcp`.
...
+syslog(LOG_LOCAL1|LOG_ERR, "Operation failed");
+closelog();
+```
+
+<--->
++++
+title=Python
++++
+
+```python
```
Comment (ricardokirkner) [inline docs/src/increase-observability/logs/forward-logs.md:153]: I'd also document the standard logging support for syslog

```
import logging
import logging.handlers

logger = logging.getLogger(__name__)
logger.setLevel(logging.DEBUG)
handler = logging.handlers.SysLogHandler(address="/dev/log")
logger.addHandler(handler)
```

File: docs/src/increase-observability/logs/forward-logs.md [hunk 3]
```diff
@@ -125,3 +125,66 @@ To include a property, add it as a flag, for example `--protocol tcp`.
 This should let you connect to any service that has syslog endpoints.
 
 To start forwarding logs, once you've added the service [trigger a redeploy](../../development/troubleshoot.md#force-a-redeploy).
+
+## Log levels
+
+Your app might output logs with distinct severity levels.
```
Comment (chadwcarlson) [inline docs/src/increase-observability/logs/forward-logs.md:131]:
```suggestion
Your app may output logs with distinct severity levels.
```

File: docs/src/increase-observability/logs/forward-logs.md (hunk 2 above)
Comment (AnouckColson) [reply docs/src/increase-observability/logs/forward-logs.md:173]: Hi @ricardokirkner, have you had a chance to look at the latest changes made by @sivachandran, and if so are you happy with them?

Comment (ricardokirkner) [reply docs/src/increase-observability/logs/forward-logs.md:173]: LGTM

**Review by ricardokirkner — DISMISSED** [review]
LGTM

File: docs/src/increase-observability/logs/forward-logs.md [hunk 4]
```diff
@@ -125,3 +125,108 @@ To include a property, add it as a flag, for example `--protocol tcp`.
...
+syslog(LOG_ERR, "Operation failed");
+closelog();
+```
+
+<--->
++++
+title=Python
++++
+
+Using logging module
```
Comment (AnouckColson) [inline docs/src/increase-observability/logs/forward-logs.md:156]:
```suggestion
Using the logging module
```

File: docs/src/increase-observability/logs/forward-logs.md [hunk 5]
```diff
@@ -125,3 +125,108 @@ To include a property, add it as a flag, for example `--protocol tcp`.
...
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+handler = logging.handlers.SysLogHandler(address="/dev/log")
+logger.addHandler(handler)
+
+logger.info("Operation started")
+logger.error("Operation failed")
+```
+
+Using syslog module
```
Comment (AnouckColson) [inline docs/src/increase-observability/logs/forward-logs.md:171]:
```suggestion
Using the Syslog module:
```

File: docs/src/increase-observability/logs/forward-logs.md [hunk 6]
```diff
@@ -125,3 +125,108 @@ To include a property, add it as a flag, for example `--protocol tcp`.
...
 To start forwarding logs, once you've added the service [trigger a redeploy](../../development/troubleshoot.md#force-a-redeploy).
+
+## Log levels
+
+Your app may output logs with distinct severity levels.
+But as Plaform.sh only reads logs from `stdout`, this distinction is lost and everything gets logged at `INFO` level.
+
+To preserve the original log level, use a language-specific syslog module/package for logging.
+
+The following example code snippets show how logs can be written to syslog:
```
Comment (AnouckColson) [inline docs/src/increase-observability/logs/forward-logs.md:136]:
```suggestion
The following example code snippets show how logs can be written to Syslog:
```

File: docs/src/increase-observability/logs/forward-logs.md [hunk 7]
```diff
@@ -125,3 +125,108 @@ To include a property, add it as a flag, for example `--protocol tcp`.
...
+syslog.syslog(syslog.LOG_ERR, "Operation failed")
+syslog.closelog()
+```
+
+<--->
++++
+title=Go
++++
+
+Using log package
```
Comment (AnouckColson) [inline docs/src/increase-observability/logs/forward-logs.md:187]:
```suggestion
Using the log package:
```

File: docs/src/increase-observability/logs/forward-logs.md [hunk 8]
```diff
@@ -125,3 +125,108 @@ To include a property, add it as a flag, for example `--protocol tcp`.
...
+		panic(err)
+	}
+
+	logger.Println("Operation started...")
+	logger.Fatalln("Operation failed")
+}
+
+```
+
+Using syslog package
```
Comment (AnouckColson) [inline docs/src/increase-observability/logs/forward-logs.md:209]:
```suggestion
Using the Syslog package:
```

**Review by AnouckColson — APPROVED** [review]

**Review by AnouckColson — APPROVED** [review]

**Pull request merged**